xcodebuild/build.go

```diff
@@ -45,13 +45,15 @@ type CommandBuilder struct {
 	isWorkspace   bool
 	scheme        string
 	configuration string
+	destination   string
 
 	// buildsetting
-	forceDevelopmentTeam              string
-	forceProvisioningProfileSpecifier string
-	forceProvisioningProfile          string
-	forceCodeSignIdentity             string
-	disableCodesign                   bool
+	forceDevelopmentTeam                string
+	forceProvisioningProfileSpecifier   string
+	forceProvisioningProfile            string
+	forceCodeSignIdentity               string
+	disableCodesign                     bool
+	disableProvisioningProfileSpecifier bool
```

Comment:
why just the specifier? why do not you use forceProvisioningProfileSpecifier with empty value?

Reply:
use forceProvisioningProfileSpecifier, forceProvisioningProfile and forceCodeSignIdentity pointers to distinguish between empty string and not set cases